datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -673,7 +673,18 @@ impl GroupedHashAggregateStream {
         let spillfile = self.runtime.disk_manager.create_tmp_file("HashAggSpill")?;
         let mut writer = IPCWriter::new(spillfile.path(), &emit.schema())?;
         // TODO: slice large `sorted` and write to multiple files in parallel
-        writer.write(&sorted)?;
+        let mut offset = 0;
+        let total_rows = sorted.num_rows();
+
+        while offset < total_rows {
+            // TODO: we could consider smaller batch size as there may be hundreds of batches
```

Comment:
this comment seems backwards, shouldn't it be 'consider *larger* batchsize` ?

But in any event, hundreds of batches seems reasonable if they are all 8k rows a piece

cc @kazuyukitanimura

Reply:
There is a typo "...batch size as there may be hundreds of batches" -> "batch size as there may be hundreds of FILES"

What I meant if we have more files spilled, it makes sense, in my head at least, to have smaller batch size as would decrease memory pressure.

Reply:
I think I was confused that this says "TODO smaller batch size" as the PR already makes the batch size smaller (caps it at the configured batch size). Given the rest of the engine processes things in the same sized batches, I don't see any reason to use a different batch size for the spill files 🤔

Reply:
I have no objection @alamb, will remove the comment